pset-functions.js

```diff
@@ -31,6 +31,70 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+const cards = [19,20,21,22]
+
+for (i = 0; i < 10; i++) {
+
+  const playerCardScore = cards[Math.floor(Math.random() * cards.length)]
+  const dealerCardScore = cards[Math.floor(Math.random() * cards.length)]
+
+  blackJack(playerCardScore,dealerCardScore)
+
+  function blackJack(playerCardScore,dealerCardScore) {
+
+    if (playerCardScore === 22 && dealerCardScore === 22 ) {
+
+      console.log(`Player:${playerCardScore}`)
+      console.log(`Dealer:${dealerCardScore}`)
+      console.log('Both 22: 0')
+    }
+
+    else if (playerCardScore === 21 && dealerCardScore === 21 ) {
+
+      console.log(`Player: ${playerCardScore}`)
+      console.log(`Dealer: ${dealerCardScore}`)
+      console.log('Both 21: 21')
+    }
+
+    if (playerCardScore === 22 || dealerCardScore === 22 ) {
+
+      //console.log('One is over 21')
+      console.log(`Player: ${playerCardScore}`)
+      console.log(`Dealer: ${dealerCardScore}`)
+
+        if (playerCardScore === 22 &&  dealerCardScore !== 22 ) {
+
+          console.log(`Dealer wins with: ${dealerCardScore}`)
+        }
+
+        if (playerCardScore !==22 &&  dealerCardScore === 22 ) {
+
+          console.log(`Player wins with:${playerCardScore}`)
+
+        }
+      }
+
```

Comment:
@thetripler One suggestion I see here you have nested conditionals. this is something you want to avoid if you can. The code works which is good but see if you can rework the conditional so it's not nested.